zerver/management/commands/email-mirror.py

```diff
@@ -129,13 +131,18 @@ def done(_):
     # type: (Any) -> None
     reactor.callLater(0, reactor.stop)
 
+py3_warning = "email-mirror is not available on python 3."
+
 def main():
     # type: () -> None
-    imap_client = protocol.ClientCreator(reactor, imap4.IMAP4Client)
-    d = imap_client.connectSSL(settings.EMAIL_GATEWAY_IMAP_SERVER, settings.EMAIL_GATEWAY_IMAP_PORT,
-                               ssl.ClientContextFactory())
-    d.addCallbacks(connected, login_failed)
-    d.addBoth(done)
+    if six.PY2:
+        imap_client = protocol.ClientCreator(reactor, imap4.IMAP4Client)
+        d = imap_client.connectSSL(settings.EMAIL_GATEWAY_IMAP_SERVER, settings.EMAIL_GATEWAY_IMAP_PORT,
+                                   ssl.ClientContextFactory())
+        d.addCallbacks(connected, login_failed)
+        d.addBoth(done)
+    else:
+        print(py3_warning)
```

Comment:
Can you write these instead as 

```
if six.PY2:
    print(py3_warning)
    return
```
So that we don't have such a larger diff / code churn?